api/v1alpha1/openapi.yaml

```
@@ -5038,4 +5141,168 @@ components:
+        - target
+      properties:
+        target:
+          $ref: '#/components/schemas/DeviceDecommissionTargetType'
+    EnrollmentRequestDetails:
+      type: object
+      required:
+        - approvedBy
+      properties:
+        approved_by:
+          type: string
+          description: The name of the approver.
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)